docs/operators-and-hooks-ref.rst

```diff
@@ -1049,8 +1049,11 @@ These integrations allow you to perform various operations using various softwar
    * - `Kubernetes <https://kubernetes.io/>`__
      - :doc:`How to use <howto/operator/kubernetes>`
      -
+     - :mod:`airflow.contrib.hooks.kubernetes_hook`
```

Comment:
This is probably not part of Kubernetes, but uses it, so it should be in a separate section. A similar situation is with Hadoop and Hive. Hive uses Hadoop, but Hive is not Hadoop.

Reply:
kubernetes_hook is a generalized hook that can be used in various operators so I think it can stay here 
for the spark_kubernetes_operator and spark_kubernetes_sensor where do you think they should reside?

Reply:
This is a difficult question, because it requires many components to make it work. After a long thought, I think it can stay here, but I will ask other people. It can be considered as part of the Kubernetes extension, so the operator and hook can be in one place.
CC: @kaxil @nuclearpinguin @potiuk